portability-types-transfer/src/main/java/org/datatransferproject/types/transfer/retry/RetryMapping.java

```diff
@@ -48,13 +49,18 @@ public RetryStrategy getStrategy() {
   }
 
   public boolean matchesThrowable(Throwable throwable) {
-    // TODO: examine entire throwable, not just toString
     String input = throwable.toString();
     for (String regex : regexes) {
       if (input.matches(regex)) {
         return true;
       }
     }
```

Comment:
leaving the previous loop seems like a micro-optimization; should we just delete it for now and add something like this (or something else) in later, if we do performance testing and find this area problematic?